hyrax/spec/views/hyrax/base/unavailable.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe 'hyrax/base/unavailable.html.erb', type: :view do
```

Comment:
Overwrites https://github.com/samvera/hyrax/blob/2.x-stable/spec/views/hyrax/base/unavailable.html.erb_spec.rb

diff for posterity

```diff
6c6
<            model_name: GenericWork.model_name)
---
>            model_name: Dataset.model_name)
31,34c31,35
<     expect(rendered).to have_content 'super cool'
<     expect(rendered).to have_content 'really deposited'
<     expect(rendered).to have_content 'parental remark'
<     expect(rendered).to have_content 'Generic Work'
---
>     expect(rendered).not_to have_content 'super cool'
>     expect(rendered).not_to have_content 'really deposited'
>     expect(rendered).not_to have_content 'parental remark'
>     expect(rendered).not_to have_content 'Generic Work'
>     expect(rendered).to eq "<!-- forbidden -->\n"
```